backend/kesaseteli/applications/templates/youth_summer_voucher_email.html

```diff
@@ -29,15 +29,12 @@ <h2>{% trans 'Kesäsetelisi, ole hyvä' %}</h2>
                     <p>
                         <strong>{{first_name}} {{last_name}}</strong><br/>
                         <strong>{% trans 'Kesäsetelinumero:' %}</strong> {{summer_voucher_serial_number}}<br/>
-                        <strong>{% trans 'Kesäsetelin summa:' %}</strong> {% trans '350€' %}<br/>
+                        <strong>{% trans 'Kesäsetelin summa:' %}</strong> {{voucher_value_with_euro_sign}}<br/>
```

Comment:
It would be much better to give the value as a variable for the translation itself. Then they would be tied together and different locales could represent it differently, for example in different order. Sure this is already in a definition table / list format, so it does not matter that much.

Reply:
Hmm, that's done already in this PR, see definition of voucher_value_with_euro_sign.

Reply:
I mean the value comes from a property, and the currency sign (€) is localized to be in front or back of the value per language.